File: server/dna.core/dna.core.service/Models/ArticleTagMapMOdel.cs
using dna.core.service.Models.Abstract;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace dna.core.service.Models
{
    public class ArticleTagMapModel : IModelBase
    {
        public int Id { get; set; }

        [Required]
        public int ArticleId { get; set; }
        [Required]
        public int TagId { get; set; }

        public virtual TagModel Tag { get; set; }

        public virtual ArticleModel Article { get; set; }
    }
}

File: server/medicore.apps/src/model/articleTagMapModel.ts
import { Type } from 'class-transformer';
import { TagModel } from './tagModel';
import { ArticleModel } from './articleModel';
import { IEntityBase } from './IEntityBase';

export class ArticleTagMapModel implements IEntityBase {
    id: number;
    articleId: number;
    tagId: number;

    //region model
    @Type(() => TagModel)
    tag: TagModel;
    @Type(() => ArticleModel)
    article: ArticleModel;

    constructor() {
        this.id = 0;
        this.articleId = 0;
        this.tagId = 0;
        this.tag = null;
        this.article = null;
    }
}

